videoprocess/vpphdr_tm.cpp

```diff
@@ -560,7 +560,9 @@ bool read_frame_to_surface(FILE *fp, VASurfaceID surface_id)
 
         src_buffer = (unsigned char*)malloc(frame_size);
         assert(src_buffer);
-        fread(src_buffer, 1, frame_size, fp);
+        do{
+            n_items = fread(src_buffer, 1, frame_size, fp);
```

Comment:
what will happen if the src file is corrupted. and full size < frame_size?

Reply:
add log to check the return value instead of read in loop.